frontend/src/muralForm/MuralForm.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import TextField from "@material-ui/core/TextField";
+import { makeStyles, createStyles, Theme } from "@material-ui/core/styles";
+import AddressSearch from "../AddressSearch/addressSearch";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    flexContainer: {
+      display: "flex",
+      flexDirection: "column",
+      alignItems: "center",
+    },
+    element: {
+      margin: theme.spacing(1),
+      width: "195px",
+    },
+  })
+);
+
+interface IMuralFormProps {}
+
+function MuralForm(props: IMuralFormProps) {
+  const styles = useStyles();
+  const currYear = new Date().getFullYear();
+
+  return (
+    <form noValidate autoComplete="off">
+      <div className={styles.flexContainer}>
+        <TextField className={styles.element} required id="name" label="Name" />
+        <TextField
+          className={styles.element}
+          required
+          id="year"
+          label="year"
+          type="number"
+          defaultValue={currYear}
+        />
+        <AddressSearch />
```

Comment:
Should `AddressSearch` take a callback prop to notify the form of its selection? Disregard if that's planned